### PR TITLE
[newman] Updates for Newman version 4.5

### PR DESCRIPTION
**Pull request by FormerPhysicist**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

***This PR updates the type definition for the options object so that it conforms with version 4.5 of newman.  The options object is documented here:  [newman api reference](https://www.npmjs.com/package/newman#api-reference)***

List of changes:

- Adding `string[]` type to `folder` option: see [here](https://github.com/postmanlabs/newman/blob/8825a0b9dd06adb1e4f970e4d0591e96594898be/lib/run/index.js#L119)
- removing `noColor` option (isn't documented in the newman api reference)
- Adding `color` option with type of `'on' | 'off' | 'auto'`: [here](https://github.com/postmanlabs/newman/blob/8825a0b9dd06adb1e4f970e4d0591e96594898be/lib/reporters/cli/index.js#L75) and then [here](https://github.com/postmanlabs/newman/blob/8825a0b9dd06adb1e4f970e4d0591e96594898be/lib/reporters/cli/cli-utils.js#L101)
- Adding two new options `workingDir` and `insecureFileRead`: see [here](https://github.com/postmanlabs/newman/blob/8825a0b9dd06adb1e4f970e4d0591e96594898be/lib/run/index.js#L157) and review the newman api reference

### CODE REVIEW

File: types/newman/index.d.ts [hunk 1]
```diff
@@ -35,7 +35,15 @@ export interface NewmanRunOptions {
      * The name or ID of the folder (ItemGroup) in the collection which would
      * be run instead of the entire collection.
      */
-    folder?: string;
+    folder?: string | string[];
```
Comment (peterblazejewicz) [inline types/newman/index.d.ts]: >  [x] Add or edit tests to reflect the change.

you've ticked this, so can we cover this in the tests?

File: types/newman/index.d.ts [hunk 2]
```diff
@@ -35,7 +35,15 @@ export interface NewmanRunOptions {
...
+    /**
+     * The path of the directory to be used as working directory.
+     */
+    workingDir?: string;
```
Comment (peterblazejewicz) [inline types/newman/index.d.ts]: >  [x] Add or edit tests to reflect the change.

ditto

File: types/newman/index.d.ts [hunk 3]
```diff
@@ -35,7 +35,15 @@ export interface NewmanRunOptions {
...
+    /**
+     * Allow reading files outside of working directory.
+     */
+    insecureFileRead?: boolean;
```
Comment (peterblazejewicz) [inline types/newman/index.d.ts]: ditto

**Comment by peterblazejewicz** [comment]
for reference, we are looking into:
https://github.com/postmanlabs/newman#api-reference

**Comment by elibarzilay** [comment]
@peterblazejewicz if it's fine can you finish the review?

**Review by peterblazejewicz — APPROVED** [review]
LGTM
@FormerPhysicist thx for the contribution!

**Pull request merged**